linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml

```diff
@@ -463,3 +478,15 @@
   when:
     - extract_ip_address_result is not defined
     - extract_ip_address_result_remote is not defined
+- debug: 
+    msg: "{{ res_def['name'] }}"
+  
+- name: "Update the ip in Inventory"
```

Comment:
How about using https://docs.ansible.com/ansible/latest/modules/virt_net_module.html ?

Reply:
@JunqiZhang0 
I agree with @abraverm . virt_net module might help

Reply:
Ok I'll change it